python/ray/_private/utils.py

```diff
@@ -1271,9 +1271,7 @@ def get_wheel_filename(
     assert py_version in ray_constants.RUNTIME_ENV_CONDA_PY_VERSIONS, py_version
 
     py_version_str = "".join(map(str, py_version))
-    if py_version_str in ["36", "37"]:
```

Comment:
This was changed. Any implications of this? @architkulkarni

Reply:
Alternatively, we could keep this - and I will basically skip all the py-311 wheel checks

Reply:
No other implications outside of this code itself.  I think some others are out of date and should be fixed too, for example it looks like `universal` is no longer part of the name of any of the wheels:

This is the list of wheels from the 2.3.1 branch
```
ray-2.3.1-cp310-cp310-manylinux2014_aarch64.whl
ray-2.3.1-cp310-cp310-manylinux2014_x86_64.whl
ray-2.3.1-cp310-cp310-win_amd64.whl
ray-2.3.1-cp311-cp311-manylinux2014_aarch64.whl
ray-2.3.1-cp311-cp311-manylinux2014_x86_64.whl
ray-2.3.1-cp36-cp36m-manylinux2014_aarch64.whl
ray-2.3.1-cp36-cp36m-manylinux2014_x86_64.whl
ray-2.3.1-cp37-cp37m-manylinux2014_aarch64.whl
ray-2.3.1-cp37-cp37m-manylinux2014_x86_64.whl
ray-2.3.1-cp37-cp37m-win_amd64.whl
ray-2.3.1-cp38-cp38-manylinux2014_aarch64.whl
ray-2.3.1-cp38-cp38-manylinux2014_x86_64.whl
ray-2.3.1-cp38-cp38-win_amd64.whl
ray-2.3.1-cp39-cp39-manylinux2014_aarch64.whl
ray-2.3.1-cp39-cp39-manylinux2014_x86_64.whl
ray-2.3.1-cp39-cp39-win_amd64.whl
ray-2.3.1+dbg-cp310-cp310-manylinux2014_x86_64.whl
ray-2.3.1+dbg-cp311-cp311-manylinux2014_x86_64.whl
ray-2.3.1+dbg-cp36-cp36m-manylinux2014_x86_64.whl
ray-2.3.1+dbg-cp37-cp37m-manylinux2014_x86_64.whl
ray-2.3.1+dbg-cp38-cp38-manylinux2014_x86_64.whl
ray-2.3.1+dbg-cp39-cp39-manylinux2014_x86_64.whl
```

Reply:
This code just needs to reflect the current wheel file formats for the current Ray commit, no other implications outside of that